src/main/java/io/confluent/connect/jdbc/sink/dialect/DbDialect.java

```diff
@@ -121,6 +138,22 @@ protected void writeColumnSpec(StringBuilder builder, SinkRecordField f) {
   }
 
   protected void formatColumnValue(StringBuilder builder, String schemaName, Map<String, String> schemaParameters, Schema.Type type, Object value) {
+    if (schemaName != null) {
+      switch (schemaName) {
+        case Decimal.LOGICAL_NAME:
+          builder.append(value);
```

Comment:
Ah so over here we are just appending to the SQL directly not binding to a `PreparedStatement`... but this is only used for formatting default values into DDL so a low surface area for issues (and unlikely that decimal default values would push precision boundaries).